ortools/algorithms/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+ENABLE_LANGUAGE(CXX)
+# ENABLE_LANGUAGE(Python)
+SET(ROOT_PROJECT ${PROJECT_NAME})
+PROJECT(${PROJECT_NAME}_algorithms)
+
+FILE(GLOB ${PROJECT_NAME}_SOURCES "*.cc")
+LIST(REMOVE_ITEM ${PROJECT_NAME}_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/hungarian_test.cc")
```

Comment:
Sure. Currently trying to allow for ExternalProject_Add for mac/windows. Also trying to figure out a way to install the python module. I had actually submitted the PR for review before merge, but I this was really quick. I'll be updating the CMake build over the next few days to address all the issues!